src/Optimizely/ProjectConfig.php

```diff
@@ -89,6 +89,11 @@ class ProjectConfig
      */
     private $_experimentIdMap;
 
+    /**
+     * @var array Associative array of experiment ID to Rollout Experiment(s) in the datafile.
+     */
+    private $_RolloutExperimentIdMap;
```

Comment:
Please delete this one, only _experimentIdMap should have all the experiments. Didn't you see variationIdMap implementation.